cms/djangoapps/contentstore/tests/test_i18n.py

```diff
@@ -223,7 +225,7 @@ def test_course_explicit_english(self):
         )
 
         self.assertContains(resp,
-                            '<h1 class="page-header">Studio Home</h1>',
+                            u'<h1 class="page-header">{} Home</h1>'.format(settings.STUDIO_SHORT_NAME),
```

Comment:
```suggestion
                            u'<h1 class="page-header">𝓢𝓽𝓾𝓭𝓲𝓸 Home</h1>',
```

Reply:
Thank you @OmarIthawi . The STUDIO_SHORT_NAME is already containing a unicode text.

Are you suggesting to hard code the similar text, so the test would fail if someone changed the environment variable in the future? or is there another logic behind that?

Reply:
Yes @shadinaif I think hard coding is good for tests.

Reply:
Done